### PR TITLE
NO-TICKET - Add litle bit more logs for parsing context

### PR DESCRIPTION
**Pull request by gslevinkov**
## Description of Work

more context on what transactions we couldn't parse
i know i know - we will have history service for that, but before that, i hope it helps and brings no harm

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/history/repository/remote/TransactionDetailsRpcRepository.kt [hunk 1]
```diff
@@ -61,7 +61,7 @@ class TransactionDetailsRpcRepository(
                     transactionDetails.addAll(parsingResult.details)
                 }
                 is ParsingResult.Error -> {
-                    Timber.i("Error on parsing transaction ${parsingResult.error.message}")
+                    Timber.e(parsingResult.error, "Error on parsing transaction")
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/history/repository/remote/TransactionDetailsRpcRepository.kt:64]: parsing error is something exceptional - it's better to have entry in sentry

File: app/src/main/java/org/p2p/wallet/root/RootActivity.kt [hunk 2]
```diff
@@ -108,7 +108,7 @@ class RootActivity :
             val findFragmentError = ClassCastException(
                 "Can't log screen open event: fragment - ${supportFragmentManager.findFragmentById(R.id.rootContainer)}"
             )
-            Timber.e(findFragmentError)
+            Timber.i(findFragmentError)
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/root/RootActivity.kt:111]: no need to create entry for such error. ClassCastException always caused by onBackPressed

File: app/src/main/java/org/p2p/wallet/common/analytics/interactor/ScreensAnalyticsInteractor.kt [hunk 3]
```diff
@@ -20,6 +21,7 @@ class ScreensAnalyticsInteractor(
         val isNewScreenOpened = newScreenName != previousScreenName
         if (isNewScreenOpened) {
             appScope.launch {
+                Timber.tag("ScreensAnalyticsInteractor",).i("logScreenOpened: $newScreenName")
```
Comment (Davran331122) [inline app/src/main/java/org/p2p/wallet/common/analytics/interactor/ScreensAnalyticsInteractor.kt:24]: You forgot to remove this ?

**Review by Davran331122 — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/common/analytics/interactor/ScreensAnalyticsInteractor.kt (hunk 3 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/common/analytics/interactor/ScreensAnalyticsInteractor.kt:24]: no, i have errors from moonpay at this moment, i want to have additional info on what screen it happens

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/common/analytics/interactor/ScreensAnalyticsInteractor.kt:24]: then you have a redundant comma

File: app/src/main/java/org/p2p/wallet/history/strategy/types/UnknownParsingStrategy.kt [hunk 4]
```diff
@@ -14,6 +15,8 @@ class UnknownParsingStrategy : TransactionParsingStrategy {
         instruction: InstructionResponse,
         transactionRoot: ConfirmedTransactionRootResponse
     ): ParsingResult {
+        Timber.i("Creating unknown transaction: type=${instruction.parsed?.type} ")
+        Timber.i("Creating unknown transaction: programId=${instruction.programId} ")
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/history/strategy/types/UnknownParsingStrategy.kt:19]: mb you'll log only one here with both params?

**Review by 1sakov — APPROVED** [review]

**Review by kabramovp2p — APPROVED** [review]

**Pull request merged**